### DISCUSSION TITLE
Auth0 redirect causes infinite loop in Phantom mobile iOS

### DISCUSSION THREAD

**Discussion by jaidharosenblatt** [phantom/docs#93] [Help & Support] [answered]
### Summary

Phantom has a bug on mobile where the app crashes. I think it has to due with our combination of using `@solana/wallet-adapter-react` with `@auth0/auth0-react`, since it sometimes will prompt a sign in with wallet for an instant before going into the infinite loop/crashing.

Our app works as expected in the following other environments
* Solflare mobile browser
* Desktop Chrome with Phantom Extension
* Desktop Safari
* Mobile Safari

### Example

https://app.bridgesplit.com/

### Steps to Reproduce

1. Visit [app.bridgesplit.com ](https://app.bridgesplit.com/) 
2. If prompted, try to log in using any credentials. Most likely, it will just be an infinite loop

### Phantom Version

23.12.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hey @jaidharosenblatt,

Thank you for writing in. Are you aware of approximately when this bug was introduced or has it always been persistent on mobile?

**Reply by jaidharosenblatt** [reply]
Thanks for follow up. It immediately stopped working once we integrated auth0 around a month ago

**Reply by adamdelphantom** [reply]
All in the phantom mobile browser: I just created a new account on app.bridgesplit.com using email/pw and associated a phantom wallet. After verifying my email, I was able to create the account and login successfully. I was able to get to the "Get Started" page without issue. 

I wasn't able to reproduce.

**Reply by jaidharosenblatt** [reply]
Strange it's working for me as well now for logging in. However, logging out leads to the same previous error that I originally described.

Can you try logging out (click on profile icon then "Log out") and seeing if you are able to reproduce

**Reply by adamdelphantom** [reply]
I logged in and out a couple of times. Still no issue. If it's helpful in your debugging we use the react native web view.

https://github.com/react-native-webview/react-native-webview

**Reply by jaidharosenblatt** [reply]
Thanks for help. I'll look into react-native-webview

**Reply by jaidharosenblatt** [reply]
Another issue I've been having is dissalowed_user_agent when trying to sign in with Google SSO. Guessing this is because Google doesn't support the web view you are using? Would be nice to support this if possible.

![image](https://github.com/phantom/docs/assets/47338031/2f2b0b90-c27b-4b1d-bf0f-fcf40e3cbae8)

**Reply by adamdelphantom** [reply, marked as answer]
@jaidharosenblatt I've created a ticket internally to look into this but I can't provide any timelines currently. My suggestion is to keep an eye on the changelog here for implementation: https://github.com/orgs/phantom/discussions/categories/announcements
